### PR TITLE
Add test case for action key on hyphen in lisp symbol

### PR DESCRIPTION
**Pull request by matsl**
Add test case for action key on hyphen in lisp symbol.

### CODE REVIEW

**Comment by matsl** [comment]
@rswgnu Added this test case for a regression I think fro some time ago. Ir was never approved but can't remember if it was because of something. Is it still valid to include or can the PR be deleted?

File: test/hmouse-drv-tests.el [hunk 1]
```diff
@@ -510,6 +508,21 @@
...
+(ert-deftest hbut-key-press-on-hyphen-in-elisp-symbol ()
+  "Key press on hyphen in elisp symbol uses smart-lisp-find-tag.
+Regression: Looked up path name '-narrow'."
+  (let* ((symbol-name "hmail:msg-narrow")
+         (el-file (make-temp-file "hypb" nil ".el" (concat "(" symbol-name ")"))))
+    (unwind-protect
+        (with-current-buffer (find-file el-file)
+          (goto-char (point-min))
+          (goto-char (1- (re-search-forward "-")))
+          (should (string= (smart-lisp-at-tag-p) symbol-name))
```
Comment (matsl) [inline test/hmouse-drv-tests.el:520]: Verifies that proper symbol is picked up by `smart-lisp-at-tag-p`. That function is used in `smart-lisp-find-tag` to look up the symbol.

File: test/hmouse-drv-tests.el [hunk 2]
```diff
@@ -510,6 +508,21 @@
...
 
+(ert-deftest hbut-key-press-on-hyphen-in-elisp-symbol ()
+  "Key press on hyphen in elisp symbol uses smart-lisp-find-tag.
+Regression: Looked up path name '-narrow'."
+  (let* ((symbol-name "hmail:msg-narrow")
+         (el-file (make-temp-file "hypb" nil ".el" (concat "(" symbol-name ")"))))
+    (unwind-protect
+        (with-current-buffer (find-file el-file)
+          (goto-char (point-min))
+          (goto-char (1- (re-search-forward "-")))
```
Comment (matsl) [inline test/hmouse-drv-tests.el:519]: What point should action key be pressed on to check the regression. Is it before the hyphen, on the hyphen or after the hyphen?

**Comment by matsl** [comment]
@rswgnu Very odd that I can't see you comments to this PR. github bug? Anyway - I have added some comments here to explain what actions I have taken to improve the test case. PTAL

**Review by rswgnu — APPROVED** [review]

**Pull request merged**